cmd/process-agent/main.go

```diff
@@ -4,11 +4,49 @@ package main
 
 import (
 	"flag"
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"log"
 	_ "net/http/pprof"
+	"strings"
 
 	"github.com/DataDog/datadog-agent/cmd/process-agent/flags"
 )
 
+func setupConfig() {
+	verb, key, value :=
+		strings.ToLower(flag.Arg(1)),
+		strings.ToLower(flag.Arg(2)),
+		flag.Arg(3)
```

Comment:
What happens if the user doesn't pass in all of the args?

Reply:
flag.Arg just returns "" if they don't pass in the args, so there won't be a nil reference error. When I add support for cobra, I will make sure that it handles errors like this.